package.json

```diff
@@ -66,6 +67,7 @@
     "yargs": "^8.0.2"
   },
   "devDependencies": {
+    "@bodhiveggie/antd-icons": "0.0.1",
```

Comment:
This is a bit curious, can't really update it without depending on you to maintain this package. Would prefer to depend on an official package or the files posted here: https://ant.design/docs/spec/download